### PR TITLE
fix(work): exchange video

### PR DESCRIPTION
**Pull request by Wandkleister**

### CODE REVIEW

File: _includes/work.liquid [hunk 1]
```diff
@@ -18,7 +18,7 @@
 			</video>
 			<video
 					class="work__video-element"
-					src="./assets/S2_240221.mp4"
+					src="./assets/S2_010321_no VW.mp4"
```
Comment (timbru31) [inline _includes/work.liquid:21]: Are these changes intentional?

**Comment by timbru31** [comment]
Shoulnd't we delete the old video?

File: _includes/work.liquid (hunk 1 above)
Comment (Wandkleister) [reply _includes/work.liquid]: Adjusted ✅

**Comment by Wandkleister** [comment]
> Shoulnd't we delete the old video?

Yes, deleted it ✅

**Review by timbru31 — COMMENTED** [review]
GitHub tells meh: +33.3 MB (420%) - was there so much change? 😲

**Review by timbru31 — APPROVED** [review]
